UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.LoadTests/LiveLogs/SCS-ENC.json

```diff
@@ -0,0 +1,3 @@
+{
+"comment":"Add FSS Batch response here"
+}
```

Comment:
Please add proper comment to clarify what content this file contains and for what purpose it get used.

Reply:
removed